miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/view/MiniAppViewHandler.kt

```diff
@@ -257,9 +257,22 @@ internal class MiniAppViewHandler(
     suspend fun createMiniAppViewFromBundle(
         miniAppInfo: MiniAppInfo,
         config: MiniAppConfig,
-        onComplete: (MiniAppDisplay?, MiniAppSdkException?) -> Unit
+        manifest: MiniAppManifest?,
+        onComplete: (MiniAppDisplay?, MiniAppSdkException?) -> Unit,
     ) {
         if (miniAppStorage.isValidMiniAppInfo(miniAppInfo.id, miniAppInfo.version.versionId)) {
+            if (manifest != null) {
+                miniAppDownloader.saveManifestForMiniAppBundle(
+                    appId = miniAppInfo.id,
+                    versionId = miniAppInfo.version.versionId,
+                    languageCode = "",
+                    manifest = manifest
+                )
+                downloadedManifestCache.storeDownloadedManifest(
+                    miniAppId = miniAppInfo.id,
+                    CachedManifest(miniAppInfo.id, manifest)
+                )
```

Comment:
Not sure why we store in two places, file also in preferences. Any reason ?

Reply:
yeah,i dont know why but it's like that in android, there are two file systems there and sometimes it check one and sometimes the other one, there are tickets for that change as well. MINI-2945

Reply:
Wow. Nice, so we need to get rid of SharedPref approach soon. Thanks @munir-rakuten . We can have this for now.